src/rpc/client.cpp

```diff
@@ -27,6 +27,8 @@ static const CRPCConvertParam vRPCConvertParams[] = {
     { "setgenerate", 0 },
     { "setgenerate", 1 },
     { "generate", 0 },
+    { "generatetoaddress", 0 },
+    { "generatetoaddress", 1 },
```

Comment:
need to remove this line (`{ "generatetoaddress", 1 },`) as only integer and boolean arguments need to be converted. trying to convert the string argument results in a JSON parsing error.